lib/taskana-core/src/test/java/acceptance/ArchitectureTest.java

```diff
@@ -153,6 +153,20 @@ void exceptionsShouldBePlacedInExceptionPackage() {
     myRule.check(importedClasses);
   }
 
+  @Test
+  void exceptionsShouldNotHaveToStringMethod() {
+    ArchRule myRule =
+        classes()
+            .that()
+            .haveSimpleNameEndingWith("Exception")
+            .and()
+            .haveSimpleNameNotStartingWith("Taskana")
+            .should()
+            .notImplement("toString()");
```

Comment:
This does not work, because `notImplement` requires a class, not a method. You have to create your own ArchCondition